docroot/themes/humsci/humsci_basic/src/scss/utilities/_lists.scss

```diff
@@ -0,0 +1,21 @@
+.hb-pill-list {
```

Comment:
I think we can make this more reusable and have an `hb-inline-list`, especially if we get rid of the link styles as suggested below. Then it isn't tied to just the pills anymore.